doc/user-guide.md

```diff
@@ -268,7 +266,7 @@ We can now invoke the Write RPC on each `node` in the configuration:
   }
 
   // Invoke Write RPC on all nodes in config
-  for _, node := range allNodesConfig.Nodes() {
+  for _, node := range allNodesConfig {
```

Comment:
Is this actually possible now? I thought only `gorums.Configuration` was a slice, not the generated `dev.Configuration`.